cmd/createorupdate/createorupdate.go

```diff
@@ -265,11 +265,21 @@ func main() {
 
 	// simulate the API call to the RP
 	entry := logrus.NewEntry(logger).WithFields(logrus.Fields{"resourceGroup": os.Getenv("RESOURCEGROUP")})
-	config := &api.PluginConfig{
+	tc := api.TestConfig{
+		RunningUnderTest:   os.Getenv("RUNNING_UNDER_TEST") != "",
+		ImageResourceGroup: os.Getenv("IMAGE_RESOURCEGROUP"),
+		ImageResourceName:  os.Getenv("IMAGE_RESOURCENAME"),
+		DeployOS:           os.Getenv("DEPLOY_OS"),
+		ImageOffer:         os.Getenv("IMAGE_OFFER"),
+		ImageVersion:       os.Getenv("IMAGE_VERSION"),
+		ORegURL:            os.Getenv("OREG_URL"),
+	}
+	config := api.PluginConfig{
```

Comment:
style nit: use &api.PluginConfig here rather than & at the point of passing to createOrUpdate